src/components/Logo.jsx

```diff
@@ -0,0 +1,14 @@
+import { Link } from "react-router-dom"
+import LogoDesktop from "../assets/logo-desktop.svg"
+import LogoMobile from "../assets/logo-mobile.svg"
+
+export const Logo = (props) => {
+    return ( 
+        <Link to="/" { ...props }>
+            <h1 className="text-2xl font-bold sm:text-3xl mr-4">
+                <img src={LogoDesktop} className="h-5 hidden lg:block" alt="NorthCoders" />
```

Comment:
This alt text isn't the most accessible - perhaps "northcoders logo, an upside-down red triangle" ?